components/driver/i2c.c

```diff
@@ -194,6 +194,12 @@ static i2c_obj_t *p_i2c_obj[I2C_NUM_MAX] = {0};
 static void i2c_isr_handler_default(void *arg);
 static void IRAM_ATTR i2c_master_cmd_begin_static(i2c_port_t i2c_num);
 static esp_err_t IRAM_ATTR i2c_hw_fsm_reset(i2c_port_t i2c_num);
+static int i2c_cmd_alive_interval[I2C_NUM_MAX] = { I2C_CMD_ALIVE_INTERVAL_TICK, I2C_CMD_ALIVE_INTERVAL_TICK };
+
+static void i2c_set_alive_interval(i2c_port_t i2c_num, int interval)
```

Comment:
this method should not be declared as `static` since it is exported via the i2c.h file (in your other PR: #7147)

Reply:
Sure, removed static qualifier.